ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/popup.view.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+import * as React from 'react'
+const Marionette = require('marionette')
+const store = require('../../../js/store.js')
+
+import PopupPreview from '../../../react-component/popup-preview'
+
+const DRAG_SENSITIVITY = 10
+
+const PopupPreviewView = Marionette.ItemView.extend({
+  selectionInterface: store,
```

Comment:
The store is specifically the selectionInterface for the workspace route, instead of setting it here allow the selectionInterface to be passed in.